include/dsn/dist/failure_detector/failure_detector.h

```diff
@@ -38,6 +38,10 @@
 # include <dsn/dist/failure_detector/fd.client.h>
 # include <dsn/dist/failure_detector/fd.server.h>
 
+namespace dsn { namespace replication {
```

Comment:
I will fix this. BTW, currently the failure_detector is somewhat naive as it won't try to switch master until current master expired. Let me reimplement a better one later.